### ISSUE TITLE
Broken Link on homepage

### ISSUE THREAD

**Issue by rsoika** [microprofile/microprofile-marketing#334]
The link to the project 'eclipse/microprofile' on the micorpfoile project overview page is broken

See: https://microprofile.io/projects/

Also the followning link is broken or has no content:
https://microprofile.io/project/eclipse/microprofile-evolution-process

![image](https://user-images.githubusercontent.com/1339124/94363687-2fc59500-00c4-11eb-8c18-e4a6c20149a7.png)

**Comment by rstjames** [comment]
@rsoika 

I check the top menu "project" link and it is working... I will ask a couple of others to check it too. 

As for https://microprofile.io/project/eclipse/microprofile-evolution-process 
It is saying this... I will look into this more this week

![Screen Shot 2020-09-28 at 7 12 13 PM](https://user-images.githubusercontent.com/12132146/94464298-96c77480-01be-11eb-98e3-cd8d7f082469.png)

Thank you for the ticket

**Comment by rsoika** [comment]
Hi Rayan

I tested it again and again. Just to give you an impression what is going on I attached a console log file form firefox and some screenshots with the Firefox Console output. I do not have the impression that the page is without bugs:

https://microprofile.io/projects/

![image](https://user-images.githubusercontent.com/1339124/94480046-009f4880-01d6-11eb-9d09-2d8c4952c249.png)


![image](https://user-images.githubusercontent.com/1339124/94480071-08f78380-01d6-11eb-937c-6063f9727229.png)


[console.log](https://github.com/eclipse/microprofile-marketing/files/5294750/console.log)


In the log file you can see that I receive a lot of time outs

    HTTP/2.0 504 Gateway Timeout 303ms

Maybe you are sitting nearer to the server?

**Comment by rsoika** [comment]
And here are some more examples what I got loaded clicking on the projects (I have a 1:3 chance to get an error or a uncomplete page loaded):

![image](https://user-images.githubusercontent.com/1339124/94480637-e31eae80-01d6-11eb-9b3f-1f2de9744855.png)

![image](https://user-images.githubusercontent.com/1339124/94480672-f16cca80-01d6-11eb-97bf-9169f9fe0a2e.png)

![image](https://user-images.githubusercontent.com/1339124/94480698-f9c50580-01d6-11eb-8626-f8f14d05d304.png)

![image](https://user-images.githubusercontent.com/1339124/94480744-03e70400-01d7-11eb-92de-7c27df5c161a.png)

**Comment by rsoika** [comment]
I have been observing this for many months now. This is just a pity that something as essential as the project homepage contains such kind of errors. This only supports those who constantly claim at with Spring everything is better :-(

**Comment by rsoika** [comment]
Maybe this can be a hint to the root of the problem.
When I try to open https://saintjames.cz/  I got this response:

![image](https://user-images.githubusercontent.com/1339124/94481343-d8184e00-01d7-11eb-8906-25158b22c2d4.png)

**Comment by rstjames** [comment]
@rsoika Yah this issue has been brought up on the community call. The project section is in need of some updates/ improvements. Last I heard someone is working on cleaning this up and seeing if we can improve this section on the website sooner than later. It is out of my skill set. I will bring it up on the community call. Or if you could join the call today you can share the issue and thoughts on it.

**Comment by aeiras** [comment]
Hola Ralph @rsoika, 

MP forum website conversation relevant to that page is here and only just started. You are most welcomed to contribute in any way you see fit there as well. https://groups.google.com/forum/?pli=1#!topic/microprofile/7xEB4PO9Zqs

**Comment by aeiras** [comment]
Hola Ralph, @rsoika, 

We have tackled the MP landing page and in 2021 the Project's page will be removed. We plan to re-write the Projects page from the work completed in 2020. Templates are in place that all projects will use to gather the information directly from the their repo. ! 

Wishing you happy holidays and thank you for your contributions to the website. Closing ticket as no work will be done on the project page since it is being removed once work starts.
